website_event_snippet_calendar/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+==========================================
+Event Calendar and List Snippet and Iframe
+==========================================
+
+When you install this addon, you will be able to drop a events calendar and
+list in any website page.
+
+Additionally, you'll have that available to embed in any ``<iframe>`` in
+``/website_event_snippet_calendar/embed``.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to any website page.
+#. Edit.
+#. Drag and drop the *Feature > Events Calendar and List* snippet.
+#. You can click on the list and use the options dropdown to add or remove
+   initial events to show.
+#. Save.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/199/10.0
+
+Known issues / Roadmap
+======================
+
+* Remove ``bootstrap-datetimepicker`` bundled library when migrating to v11.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/event/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+  * Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
Add an extra line as this is not correctly rendered

Reply:
Done

Reply:
It should be before, not after